### DISCUSSION TITLE
Can't link to created pages

### DISCUSSION THREAD

**Discussion by MikeXYZ** [GetPublii/Publii#1687] [Help]
I can't get the link editor to work.  The settings I apply (page or post) don't stick. It keeps reverting to 'custom' instead of 'page' or 'post' which I apply. 
Would appreciate any advice.
TNX

**Comment by MikeXYZ** [comment]
Here is what does seem to be applied (from the rendered page copying the link);
file:///C:/Users/mikes/Documents/Publii/sites/mikes-real-estate-photography/preview/preview.html#INTERNAL_LINK#/page/5
But clicking on the link has no effect.

**Comment by MikeXYZ** [comment]
OK, if I resave the 'theme' everything works. Do we need to resave the theme after any changes made?

**Reply by bjazmoore** [reply]
> OK, if I resave the 'theme' everything works. Do we need to resave the theme after any changes made?

Do you mean by "resave the theme" that you are in Site settings -> Basic settings and you have selected the theme name from the drop-down menu under the "USE" option like in the image?  If that is what you are talking about - then you should only need to do that once.  Can you confirm you are using version 0.46.1?

![1](https://github.com/user-attachments/assets/76ab4161-8115-4163-9274-6fa952dbae8f)

**Comment by MikeXYZ** [comment]
Thanks,
What I have noticed is that when I hit the 'preview' button for a single page that I have worked on, It renders that page (in browser) but any links on that preview do not function.
But if I hit 'preview changes made' on the main menu (left side of the screen) it throws up a functional site preview where links to other of my pages DO work.
  All sorted.
In the meantime I have found that I can run an old copy of Macromedia Dreamweaver 8 in compatibility mode on Win 11. I'm reasonably familiar with that program, so using it will obviate a steep learning curve and th need for paid plugins to do tricky things using Publii.
That's the plan anyway. If it doesnt work out, I'll be back to Publii.
Thanks again for your help
